core/src/main/kotlin/entity/application/ApplicationCommand.kt

```diff
@@ -66,7 +66,7 @@ public interface GlobalApplicationCommand : ApplicationCommand, GlobalApplicatio
     /**
      * Whether this command is available in DMs with the application.
      */
-    public val dmPermission: Boolean get() = data.dmPermission?.value ?: true
+    public val dmPermission: Boolean get() = data.dmPermission.orElse(true)
```

Comment:
I guess this should be or else false cause if it's absent it means it's not permited in Dms?

Reply:
[This table](https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-structure) suggest the opposite:
> By default, commands are visible.